tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+import fnmatch
+import os
+
+import nengo
+from nengo.tests.conftest import *
+from nengo.utils.testing import find_modules, allclose, load_functions
+
+import nengo_spinnaker
+
+
+# A monkey patch hack to solve a problem in Nengo master
+def get_filename(self, ext=''):
+    modparts = self.module_name.split('.')[1:]
+    return "%s.%s.%s" % ('.'.join(modparts), self.function_name, ext)
+nengo.utils.testing.Recorder.get_filename = get_filename
+
+
+@pytest.fixture
+def Simulator():
+    return nengo_spinnaker.Simulator
```

Comment:
This was my original intention; here's what I had in my first draft but it didn't work:
```python
@pytest.fixture
def Simulator(request):
    def _simulator(*args, **kwargs):
        sim = nengo_spinnaker.Simulator(*args, **kwargs)
        request.addfinalizer(sim.close)
        return sim
    return _simulator
```
But, that was before I'd actually looked at `simulator.py`, so I realize now that I can just use `_close_open_simulators` as a finalizer. I'll try that out now.